test/unit/materialized_view_identifier_test.sql

```diff
@@ -0,0 +1,88 @@
+select has_column('ggircs_swrs', 'identifier', 'swrs_facility_id', 'ggircs_swrs.identifier has column: swrs_facility_id');
+select has_column('ggircs_swrs', 'identifier', 'identifier_type', 'ggircs_swrs.identifier has column: identifier_type');
+select has_column('ggircs_swrs', 'identifier', 'identifier_value', 'ggircs_swrs.identifier has column: identifier_value');
+select has_column('ggircs_swrs', 'identifier', 'swrs_identifier_history_id', 'ggircs_swrs.identifier has column: swrs_identifier_history_id');
+
+-- Test index names in matview report exist and are correct
+select has_index('ggircs_swrs', 'identifier', 'ggircs_identifier_primary_key', 'ggircs_swrs.identifier has index: ggircs_identifier_primary_key');
+select has_index('ggircs_swrs', 'identifier', 'ggircs_swrs_identifier_history', 'ggircs_swrs.identifier has index: ggircs_swrs_identifier_history');
+
+-- Test unique indicies are defined unique
+select index_is_unique('ggircs_swrs', 'facility', 'ggircs_facility_primary_key', 'Matview report index ggircs_facility_primary_key is unique');
+
+-- Test columns in matview report have correct types
+select col_type_is('ggircs_swrs', 'identifier', 'id', 'bigint', 'ggircs_swrs.identifier column id has type bigint');
+select col_type_is('ggircs_swrs', 'identifier', 'report_id', 'bigint', 'ggircs_swrs.identifier column report_id has type bigint');
+select col_type_is('ggircs_swrs', 'identifier', 'swrs_report_id', 'numeric(1000,0)', 'ggircs_swrs.identifier column swrs_report_id has type numeric');
+select col_type_is('ggircs_swrs', 'identifier', 'facility_id', 'bigint', 'ggircs_swrs.identifier column facility_id has type bigint');
+select col_type_is('ggircs_swrs', 'identifier', 'swrs_facility_id', 'numeric(1000,0)', 'ggircs_swrs.identifier column swrs_facility_id has type numeric');
+select col_type_is('ggircs_swrs', 'identifier', 'identifier_type', 'character varying(1000)', 'ggircs_swrs.identifier column identifier_type has type varchar');
+select col_type_is('ggircs_swrs', 'identifier', 'identifier_value', 'character varying(1000)', 'ggircs_swrs.identifier column identifier_value has type varchar');
+select col_type_is('ggircs_swrs', 'identifier', 'swrs_identifier_history_id', 'bigint', 'ggircs_swrs.identifier column swrs_identifier_history_id has type bigint');
+
+
+-- insert necessary data into table ghgr_import
+insert into ggircs_swrs.ghgr_import (xml_file) values ($$
+    <ReportData xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+      <RegistrationData>
+        <Facility>
+          <Identifiers>
+            <IdentifierList>
+              <Identifier>
+                <IdentifierType>GHGRP Identification Number</IdentifierType>
+                <IdentifierValue>R0B0T2</IdentifierValue>
+              </Identifier>
+            </IdentifierList>
+          </Identifiers>
+        </Facility>
+      </RegistrationData>
+      <ReportDetails>
```

Comment:
by removing the transitive dependencies on other views, hopefully we'll be able to reduce the size of the fixture required to test this